core/controllers/dashboard.py

```diff
@@ -190,8 +195,33 @@ def _get_intro_card_color(category):
             key=lambda x: (x['num_open_threads'], x['last_updated']),
             reverse=True)
 
+        for collection_summary in subscribed_collection_summaries:
+            if collection_summary is None:
+                continue
+
+            collections_list.append({
```

Comment:
Going to leave this one for @526avijitgupta for later -- don't want to do this sort of change just before release. Will add a TODO.